app/plugin_install.go

```diff
@@ -290,6 +312,42 @@ func (a *App) removePluginLocally(id string) *model.AppError {
 	return nil
 }
 
+func (a *App) saveSignatures(pluginId string, signatures []string) *model.AppError {
+	for counter, signature := range signatures {
+		signatureBytes, err := base64.StdEncoding.DecodeString(signature)
```

Comment:
this works, however I'm thinking would it be a tiny bit nicer if 
`InstallPluginWithSignatures` accepted `signatures []io.Readers` as opposed to `[]string`? The reason I say this is the base64 encoding/decoding is only a concern for the marketplace server and this server, it shouldn't be a concern for how we install/save the signatures in our filestore. This will reduce the complexity on how we save/verify signatures in this file. Thoughts?